data.json

```diff
@@ -1,6 +1,7 @@
 [
     {
         "name": "WILLIANA JONES BURROUGHS",
+        "placeId": "ChIJIxM9EXH2wokR3w5eQWsSV44",
```

Comment:
this is what i was thinking too. i think we should store placeId's in the db when we store the other info. this just made me realize that we're actually not using this file anymore since we're pulling directly from the db. i'll add the placeIds to the db and delete this file in the next go around.

Reply:
yeah, I realized we weren't using it but figured it was still worth updating so someone could use it to update Firebase so that's perfect